common/src/main/scala/com/lightbend/rp/common/SocketBinding.scala

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright 2017 Lightbend, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.lightbend.rp.common
+
+import scala.collection.immutable.Seq
+import scala.collection.JavaConverters._
+
+object SocketBinding {
+  private val reader = new EnvironmentReader(System.getenv().asScala.toMap)
```

Comment:
I think we can use `sys.env` in place of `System.getenv().asScala.toMap`.

Reply:
That's certainly easier!